firefox-webext/background.js

```diff
@@ -5,6 +5,26 @@
 var prefix = 'https://webcompat.com/?open=1&url=';
 var screenshotData = '';
 
+chrome.contextMenus.create({
+  id: "webcompat",
```

Comment:
`id` is optional, so I'd recommend either leaving it out, or picking something more specific like `webcompat-contextmenu`.